src/mainwindow.cpp

```diff
@@ -479,6 +487,12 @@ void PointViewerMainWindow::addFiles()
     m_currFileDir.makeAbsolute();
 }
 
+void PointViewerMainWindow::finalSetup()
+{
+    openShaderFile("shaders:las_points.glsl");
```

Comment:
Ok, done - have a look if you are ok with calling the parent window, otherwise more stuff has to be moved into View3D.